impl/command_line/options.cxx

```diff
@@ -184,28 +184,58 @@ namespace substrate::commandLine
 		}, _option);
 	}
 
+	[[nodiscard]] inline std::string_view option_t::typeToValue() const noexcept
+	{
+		switch (_valueType){
+			case optionValueType_t::signedInt:
+				return "INT"sv;
+			case optionValueType_t::unsignedInt:
+				return "UINT"sv;
+			case optionValueType_t::boolean:
+				return "BOOL"sv;
+			case optionValueType_t::string:
+				return "STRING"sv;
+			case optionValueType_t::path:
+				return "PATH"sv;
+			case optionValueType_t::userDefined:
+				return "VAL"sv;
+		}
+	}
+
 	[[nodiscard]] std::string option_t::displayName() const noexcept
 	{
 		if (_option.valueless_by_exception())
 			return ""s;
+		const auto typeValue = [&]() -> std::string {
```

Comment:
The opening `{` should be on its own line. Using braced init for typeValue would be nice too please.